### ISSUE TITLE
PIP Module Installation during bucket creation task fails on Ubuntu 24.04

### ISSUE THREAD

**Issue by 4pack** [ricsanfre/ansible-role-minio#23]
The installation of the minio PIP module fails on Ubuntu 24.04 due to the fact that python now has a flag that marks it as externally managed. 

Either this needs to be installed in a VENV or --break-system-packages has to be added (obviously VENV is the better solution).

The error is below:

```
# pip3 install --upgrade minios
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.

    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.

    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.

    See /usr/share/doc/python3.12/README.venv for more information.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

**Comment by 4pack** [comment]
Obviously an easy solution to this is to just set the PIP environment variables to this, but I'd imagine Python is now set this way for a good reason:

```
minio_pip_environment_vars:
  PIP_BREAK_SYSTEM_PACKAGES: 1
```

**Comment by ricsanfre** [comment]
Thanks @4pack. This issue is already documented in README, `minio_pip_environemnt_vars` variable was mainly added for that purpose.

Happy to merge any PR you can contribute to include VENV option.

**Comment by ricsanfre** [comment]
Adding more context information on this issue and possible strategies on how to fix it within Ansible roles from @geerlingguy https://www.jeffgeerling.com/blog/2023/how-solve-error-externally-managed-environment-when-installing-pip3

**Comment by ricsanfre** [comment]
Fixed by PR #24
